algorithm/all/all.go

```diff
@@ -0,0 +1,7 @@
+package all
```

Comment:
maybe move this file to  `algorithm/all.go`?

Reply:
can't do it, will be cyclic import